README.md

```diff
@@ -15,7 +15,7 @@ Here is a **curated selection** of the metrics, tools and statistical tests incl
 |-----------------------	|-----------------	|--------------	|
 | **[Continuous](https://scores.readthedocs.io/en/latest/included.html#continuous)**        	|Scores for evaluating single-valued continuous forecasts.                  	|Mean Absolute Error (MAE), Mean Squared Error (MSE), Root Mean Squared Error (RMSE), Additive Bias, Multiplicative Bias, Pearson's Correlation Coefficient, Flip-Flop Index, Quantile loss, Murphy score.              	|
 | **[Probability](https://scores.readthedocs.io/en/latest/included.html#probability)**       	|Scores for evaluating forecasts that are expressed as predictive distributions, ensembles, and probabilities of binary events.                 	|Brier Score, Continuous Ranked Probability Score (CRPS) for Cumulative Density Function (CDF), Threshold weighted CRPS for CDF, CRPS for ensembles, Receiver Operating Characteristic (ROC), Isotonic Regression (reliability diagrams).              	|
-| **[Categorical](https://scores.readthedocs.io/en/latest/included.html#categorical)**       	|Scores for evaluating forecasts based on categories.                	|Probability of Detection (POD), False Alarm Rate, Probability of False Detection (POFD), Success Ratio, Accuracy, Peirce's Skill Score, Critical Success Index (CSI), Gilbert Skill Score, Heidke Skill Score, Odds Ratio, Odds Ratio Skill Score, F1 score, FIxed Risk Multicategorical (FIRM) Score.               	|
+| **[Categorical](https://scores.readthedocs.io/en/latest/included.html#categorical)**       	|Scores for evaluating forecasts based on categories.                	|Probability of Detection (POD), False Alarm Rate, Probability of False Detection (POFD), Success Ratio, Accuracy, Peirce's Skill Score, Critical Success Index (CSI), Gilbert Skill Score, Heidke Skill Score, Odds Ratio, Odds Ratio Skill Score, F1 score, Symmetric Extremal Dependence Index, FIxed Risk Multicategorical (FIRM) Score.               	|
```

Comment:
Yes - it's a widely used metric (paper cited 231 times). I'd be happy to shorten it its acronym - I'm actually happy shorten more of these to their acronyms if needed.

Reply:
I know this isn't directly related to the pull request - but in terms of the table length: is it necessary to include both "False Alarm Rate" and "Probability of False Detection (POFD)" - perhaps only one needs to be included? (My guess is the double inclusion wasn't intentional, and occurred by accident).

Reply:
It should actually be "False Alarm *Ratio*" which is different. This typo needs fixing in the paper too.

Reply:
I've updated the table in this pull request

Reply:
Does that mean included.md needs updating too?

Reply:
Sorry - I think I misunderstood. I thought you meant False Alarm Ratio is the same as POFD (in which case the included.md table needs updating, as it says False Alarm Rate is the same as POFD). But now I think you meant "False Alarm Rate" in the readme and paper table should have read False Alarm Ratio. If I am now understanding correctly (?), then I think the table in included.md is correct?

Reply:
I think Nick means: This ReadMe table should include false alarm Ratio. POFD = False Alarm Rate (see https://www.cawcr.gov.au/projects/verification/) . The reasoning being that it's redundant to list both False Alarm Rate and POFD, so it's better to list a different things rather than double up with a synonymous metric. So we have correctly variously said False Alarm Rate is the same thing as probability of false detection . However, for the purposes of a more interesting table on the Readme, we should omit False Alarm Rate and include False Alarm Ratio in its place. Included.md doesn't need changing. I think he's saying the paper should be updated on the same principle of making the table more diverse.

Reply:
That was my interpretation on re-reading Nick's comment. So it sounds like I am no longer confused :-).

Re. the paper - the paper was previously changed to say "Probability of Detection (POD), False Alarm Rate , Success Ratio,". So instead of changing "False Alarm Rate" to "False Alarm Ratio", "False Alarm Rate" was instead kept and "POFD" was deleted. I don't think False Alarm Ratio is included in the paper at the moment.

Reply:
Yes, Tennessee's comment above matches what I meant. I think False Alarm Ratio should probably be included in the paper as it's widely used, but that discussion can happen on that pull request rather than here.